### PR TITLE
Add 500px to the social networks list

### PR DESCRIPTION
**Pull request by salvoventura**
I added support for 500px which I find useful for photographers.

### CODE REVIEW

**Comment by jonathanbardo** [comment]
@salvoventura Thanks! Can you move this right after or before Flickr?

File: includes/social-networks.php [hunk 1]
```diff
@@ -12,6 +12,12 @@
 $board    = esc_attr_x( 'board', 'Must be lowercase and use url-safe characters', 'contact-widgets' );
 
 $fields = [
+	'500px'    => [
+		//'icon'    => 'facebook-official',
```
Comment (jonathanbardo) [inline includes/social-networks.php]: @salvoventura Please remove this line as well.

Comment (salvoventura) [inline includes/social-networks.php]: Sure, I will. Should I change it in my branch and submit a new pull request?

Comment (jonathanbardo) [inline includes/social-networks.php]: You can update your master branch and it will be reflected here.

Comment (salvoventura) [inline includes/social-networks.php]: Done. Thanks Jonathan!

**Comment by jonathanbardo** [comment]
@salvoventura Actually, one last thing. Can you re-issue this pull request against the develop branch? The master branch is for release only. Thank you!

**Comment by salvoventura** [comment]
Done! Sorry, catching up with the workflow... I'll somehow have to close the pull request on the master branch now I guess.

**Comment by jonathanbardo** [comment]
Closing in favor of #13

**Pull request closed without merging**